packages/js/email-editor/src/components/sidebar/template-settings-panel.tsx

```diff
@@ -20,10 +22,14 @@ export function TemplateSettingsPanel() {
 	}
 
 	return (
-		<>
+		<PluginDocumentSettingPanel
+			name="template-settings-panel"
+			title={ __( 'Settings', 'woocommerce' ) }
+			className="woocommerce-email-editor__settings-panel"
+		>
 			{ templateSections.map( ( section ) => (
 				<div key={ section.id }>{ section.render() }</div>
 			) ) }
-		</>
+		</PluginDocumentSettingPanel>
```

Comment:
I think at this point, we should wrap this component in a custom error boundary so that errors that occur from third-party integration do not totally break the whole email editor interface.